migrate/repo_checks.py

```diff
@@ -78,6 +97,253 @@ def dry_run(self):
+        # Get the content of the .github files, maybe this should be a memoized
+        # function since we'll want to get the same .github content from all
+        # the repos.
+        for file in self.workflow_templates:
+            file_path = f"workflow-templates/{file}"
+            try:
+                self.dot_github_template_contents[file] = get_github_file_contents(
+                    self.api,
+                    self.org_name,
+                    ".github",
+                    file_path,
+                    dot_github_default_branch,
+                )
+            except HTTP4xxClientError as e:
+                click.echo(
+                    f"File: https://github.com/{org_name}/.github/blob/{dot_github_default_branch}/{file_path}"
+                )
+                click.echo(e.fp.read().decode("utf-8"))
+                raise
+
+        # Get the content of the repo specific file.
+        repo_contents = {}
+        files_that_are_missing = []
+        for file in self.workflow_templates:
+            file_path = f".github/workflows/{file}"
+            try:
+                repo_contents[file] = get_github_file_contents(
+                    self.api,
+                    self.org_name,
+                    self.repo_name,
+                    file_path,
+                    branch_name,
+                )
+            except HTTP4xxClientError as e:
+                if e.status == 404:
+                    files_that_are_missing.append(file)
+
+        # Compare the two.
+        files_that_differ = []
+        for file in self.workflow_templates:
+            if (
+                file not in files_that_are_missing
+                and self.dot_github_template_contents[file] != repo_contents[file]
+            ):
+                files_that_differ.append(file)
+
+        return (files_that_differ, files_that_are_missing)
+
+    def dry_run(self):
+        return self.fix(dry_run=True)
+
+    def fix(self, dry_run=False):
+        """
+        Always use the same branch name and update the contents if necessary.
+        """
+
+        steps = []
+        # Check to see if the update branch already exists.
+        branch_exists = True
+        try:
+            self.api.git.get_ref(
+                self.org_name, self.repo_name, f"heads/{self.branch_name}"
+            )
+        except HTTP4xxClientError as e:
+            if e.status == 404:
+                branch_exists = False
+            else:
+                raise  # For any other unexpected errors.
+
+        # Get the hash of the default branch.
+        repo = self.api.repos.get(self.org_name, self.repo_name)
+        default_branch = repo.default_branch
+        default_branch_sha = self.api.git.get_ref(
+            self.org_name,
+            self.repo_name,
+            f"heads/{default_branch}",
+        ).object.sha
+
+        if branch_exists:
+            steps.append("Workflow branch already exists.  Updating branch.")
```

Comment:
You include the update branch name in the 'Creating' log line below and I think that's just great.
```suggestion
            steps.append(f"Workflow branch already exists.  Updating '{self.branch_name}'.")
```